views/pages/services/student-services.pug

```diff
@@ -0,0 +1,54 @@
+extends /views/templates/frontend-base.pug
+include /views/mixins/pink-button.pug
+
+append styles
+    link(href="/assets/css/pages/services/student-services.css" rel="stylesheet")  
+    link(rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css")
+
+block body 
+    div(class="service")
+        div(class="service-item")
+            h2 Locker Sign Out 
+            p If a student in the math faculty is on a study term and has paid their MathSoc fee, they are eligible to book a locker for the period of the term. No perishable foods are allowed to be stored to avoid attracting pests. MathSoc offers lockers on a term-by-term basis. Locker sign ups are closed for the term.
+        img(src="/assets/img/services/lockers.jpg", alt="lockers", class="service-item")
+    div(class="service reverse")
+        div(class="reverse-item service-item")
+            h2 Board Game Nights 
+            p MathSoc holds weekly Board Game Nights for students to unwind and play board games with their friends. Stop by the office to pick your game of choice from MathSoc’s extensive collection!
+            p Some titles we have in store: 
+            ul
+                li Settlers of Catan 
+                li Jenga
+                li  Dominion 
+                li Shadows Over Camelot 
+                li  Deception 
+                li ...and many more! 
+            p Join the MathSoc Games Night Discord to recieve more updates on events: 
+            a(href="https://discord.gg/DAT6S2TWdB") https://discord.gg/DAT6S2TWdB. 
+        img(src="/assets/img/services/board-game-night.jpg", alt="board games on shelves", class="reverse_item service-item")
+    div(class="service")
+        div(class="service-item") 
+            h2 Math Cnd 
+            p Math CnD is owned and operated by the Math Society and located on the 3rd floor in MC Building. CnD has a great selection of food choices at student friendly prices such as coffee & donuts, hot soup and entrees, cold sandwiches, Wasabi Sushi, fresh patties and samosas, and so much more. It’s worth the trip to MC!!
+            p Don’t forget our daily specials, which are always in a limited supply:
+            ul 
+                li Monday - Mac 'n Cheese
+                li Tuesday - Soup (Creamy Potato With Bacon always available)
+                li Wednesday - Chicken Stew 
+                li Thursday - Soup (Boston Clam Chowder always available)
+                li Friday - Chili and Garlic Bread Stix (sold separately)
+            p The Store Manager Rose Penner can be reached at: 519-888-4567 ext. 36373 or at 
+            | <a href="mailto:r2penner@uwaterloo.ca">r2penner@uwaterloo.ca</a>. 
+            | The CnD can be reached directly at 519-888-4567 ext. 36521.    
```

Comment:
Two things:
* Can we add indentation so that the entire text is included as part of the `p` tag?
* Is there a reason not to use pug for the `a` tag?  We should try to use pug where possible!

```suggestion
            p The Store Manager Rose Penner can be reached at: 519-888-4567 ext. 36373 or at 
              | a(href="mailto:r2penner@uwaterloo.ca") r2penner@uwaterloo.ca. 
              | The CnD can be reached directly at 519-888-4567 ext. 36521.    
```